store/mongo/datastore_mongo_releases.go

```diff
@@ -219,3 +220,60 @@ func (db *DataStoreMongo) UpdateRelease(
 	}
 	return nil
 }
+
+// Save the possibly new update types
+func (db *DataStoreMongo) SaveUpdateTypes(ctx context.Context, updateTypes []string) error {
+	database := db.client.Database(DatabaseName)
+	c := database.Collection(CollectionUpdateTypes)
+
+	tenantId := ""
+	if id := identity.FromContext(ctx); id != nil {
+		tenantId = id.Tenant
+	}
+	options := mopts.UpdateOptions{}
+	options.SetUpsert(true)
+	_, err := c.UpdateOne(
+		ctx,
+		bson.M{
+			StorageKeyId:       StorageKeyStorageReleaseUpdateTypes,
+			StorageKeyTenantId: tenantId,
+		},
+		bson.M{
+			"$addToSet": bson.M{
+				StorageKeyStorageReleaseUpdateTypes: bson.M{
+					"$each": updateTypes,
+				},
+			},
+		},
+		&options,
+	)
+	return err
+}
+
+// Get the update types
+func (db *DataStoreMongo) GetUpdateTypes(ctx context.Context) ([]string, error) {
+	database := db.client.Database(DatabaseName)
+	c := database.Collection(CollectionUpdateTypes)
+
+	tenantId := ""
+	if id := identity.FromContext(ctx); id != nil {
+		tenantId = id.Tenant
+	}
+	result := c.FindOne(
+		ctx,
+		bson.M{
+			StorageKeyId:       StorageKeyStorageReleaseUpdateTypes,
+			StorageKeyTenantId: tenantId,
```

Comment:
shall I create an index for that?

Reply:
Why not using the tenantId as `_id`? This way you don't need to create a new index.
The `StorageKeyStorageReleaseUpdateTypes` as _id is useless.

Reply:
> Why not using the tenantId as `_id`? This way you don't need to create a new index. The `StorageKeyStorageReleaseUpdateTypes` as _id is useless.

I did not like the idea that tenant_id it can be empty. the `StorageKeyStorageReleaseUpdateTypes` is not that bad, mongo does use this schema to refer to "singular objects in a collection". anyways, if you want me to move to tenant_id what do we do with the case when tenant_id is empty?

Reply:
Ok, makes sense.